docs/source/community-study-approval-process.rst

```diff
@@ -80,20 +120,21 @@ When your study is ready and you’ve responded to the peer feedback,
 you can :ref:`submit your study <study status>` for Lookit approval so it can go live.
 This is the point where you'll need to have your access agreement set up.
 
-When you submit the study, you will be prompted to note any non-standard elements that require specific approval per the Terms of Use (e.g., integration of
+When you submit the study, you will be prompted to note any non-standard elements 
+that require specific approval per the Terms of Use (e.g., integration of
 additional about participants from another source) as well as what
```

Comment:
should this be additional "information"?

Reply:
indeed! fixed